src/components/PricingCard/PricingCardAttributeItem.react.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+import Icon from "../Icon";
+// TODO: Add RootComponent prop
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +included?: boolean,
```

Comment:
Let's rename included to something a little more explanatory

Reply:
I've updated component and I thought **available** is the most suitable name , but we can think any better prop name